2006Scape Client/src/main/java/Main.java

```diff
@@ -35,10 +35,36 @@ public static void main(String[] args) {
 						Game.customSettingVisiblePlayerNames = true;
 						Game.customSettingMinItemValue = 1000;
 						break;
+					case "-fixesnooverride":
```

Comment:
whats the purpose of this?

Reply:
I wanted a way to add some qol options, and then I wanted a way to add qol options without forcing settings like the min item value and visible player names. It's not necessarily the cleanest code ever, but I don't think it's anything worse than currently exists.

Reply:
i dont mind removing the min item value & player names for the sake of code cleanliness cause they can be changed in the custom settings tab thats enabled with -qol anyway